src/guide/essentials/template-refs.md

```diff
@@ -1,25 +1,26 @@
 # Template Refs
 
-While Vue's declarative rendering model abstracts away most of the direct DOM operations for you, there may still be cases where we need direct access to the underlying DOM elements. To achieve this, we can use the special `ref` attribute:
+Vue ning deklarativ renderlash modeli(template sintaksisi( `{}` ) orqali ma'lumotlarni DOMga chiqarish) siz uchun DOM ning to'g'ridan-to'g'ri bajariladigan ko'p operatsiyalarini(Masalan, `document.querySelector()`) qisqartirib bergan bo'lsa-da, DOM 
```

Comment:
bu yerda masalan so'zidan oldin qavs bor, va usha qavsdan oldin probel, joy tashlab ketish kerak
![image](https://user-images.githubusercontent.com/12369295/172146510-6a564182-752a-42ce-a98a-1808a05890d7.png)

Reply:
To'g'rilandi.